common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -37,6 +38,7 @@ case class DotcomRenderingDataModel(
     webTitle: String,
     mainMediaElements: List[PageElement],
     main: String,
+    activeTopic: Option[String],
     filterKeyEvents: Boolean,
     topics: Option[Seq[TopicWithCount]],
```

Comment:
also maybe we could have "availableTopics" and "selectedTopics" sit next to each other in the model for an easier read

Reply:
done 👍